app/src/main/res/values/dimens.xml

```diff
@@ -112,9 +130,10 @@
     <dimen name="tag_item_using_size">10sp</dimen>
     <dimen name="tag_item_icon_size">14dp</dimen>
     <dimen name="tag_item_icon_padding">1dp</dimen>
+    <dimen name="tag_item_margin_side">22dp</dimen>
```

Comment:
there is an **tag_item_margin_side** twice ... can you delete one? probably a rebase left-over